Cargo.toml

```diff
@@ -1,11 +1,14 @@
 [package]
+categories = ["command-line-utilities"]
+description = "Gets the time somebody worked on something from `git log`."
+edition = "2021"
+keywords = ["cli", "git", "summary", "time"]
+license = "MIT"
 name = "commit-analyzer"
+repository = "https://github.com/wert007/commit-analyzer"
 version = "0.1.0"
-edition = "2021"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-chrono = "*"
-getopts = "*"
-sysexits = "*"
+chrono = "0.4.19"
```

Comment:
`cargo clippy -- -D clippy::cargo` parses the manifest and suggests optimisations.  Naming the dependency versions explicitly is one of them.

People usually look at the `Cargo.toml`, at first, before searching for the `Cargo.lock`, if any.  In case a certain version of a crate should break their Rust environment, they can figure it out that way much easier.

Reply:
Hmm, interesting. I guess that makes sense.